### PR TITLE
Fix having ds annotations on path.

### PR DESCRIPTION
**Pull request by akurtakov**
Ignore generated file. Continuation for  #984

### CODE REVIEW

File: tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties [hunk 1]
```diff
@@ -15,3 +15,4 @@ bin.includes = META-INF/,\
                about.html,\
                plugin.properties
 source.. = src/main/java/
+additional.bundles = org.eclipse.pde.ds.lib
```
Comment (laeubi) [inline tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties:18]: this should really not be nessesary

File: tycho-bundles/org.eclipse.tycho.p2.resolver.impl/OSGI-INF/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/org.eclipse.tycho.p2.manager.ReactorRepositoryManagerImpl.xml
```
Comment (laeubi) [inline tycho-bundles/org.eclipse.tycho.p2.resolver.impl/OSGI-INF/.gitignore:1]: yes we can ignore this file as it is automatically generated

File: tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties (hunk 1 above)
Comment (laeubi) [reply tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties:18]: If the DS builder is enabled, it should add this automatically here and no warning should be shown.

Comment (akurtakov) [reply tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties:18]: OK. I'll still push this one so we unbreak things for now. Once m2e/pde/whatever gets fixed let's remove it.

Comment (laeubi) [reply tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties:18]: You can better then add an import package (with optional) instead.

Comment (akurtakov) [reply tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties]: Let's see what happens.

**Pull request merged**